src/VSCodeExtension/src/commands.ts

```diff
@@ -22,13 +22,76 @@ export function registerCommand(context: vscode.ExtensionContext, name: string,
     )
 }
 
+function createNewProjectAtUri(dotNetSdk: DotnetInfo, projectType: string, uri: vscode.Uri) {
+    let proc = cp.spawn(
+        dotNetSdk.path,
+        ["new", projectType, "-lang", "Q#", "-o", uri.fsPath]
+    );
+    
+    let errorMessage = "";
+    proc.stderr.on('data', data => {
+        errorMessage = errorMessage + data;
+    });
+
+    proc.on(
+        'exit', (code, signal) => {
+            if (code === 0) {
+                const openItem = "Open new project...";
+                vscode.window.showInformationMessage(
+                    `Successfully created new project at ${uri.fsPath}.`,
+                    openItem
+                ).then(
+                    (item) => {
+                        if (item === openItem) {
+                            vscode.commands.executeCommand(
+                                "vscode.openFolder",
+                                uri
+                            ).then(
+                                (value) => {},
+                                (value) => {
+                                    vscode.window.showErrorMessage("Could not open new project");
+                                }
+                            );
+                        }
+                    }
+                );
+            } else {
+                // Check if the problem was that the project templates are missing.
+                // If so, we can give a more helpful error message and offer the
+                // user to install templates.
+                if (errorMessage.includes("'Q#' is not a valid value for -lang")) {
```

Comment:
If someone has a different language setting, I would not expect that this is in English? I think it should be fine here to just look for Q# only, and modify the text below to say
"... the Q# project templates may not be installed. "
```suggestion
                if (errorMessage.includes("Q#") && errorMessage.includes("-lang")) {
```